features/step_definitions/create_menu_page.rb

```diff
@@ -16,8 +16,17 @@
       path = new_user_registration_path
     when 'restaurants' then
       path = new_restaurant_path
+    when 'create dish' then
```

Comment:
Okay then if you have a chore to refactor the step defs.. In general the less code we pull in the first place the less we have to refactor later on. But I will let this one fly for now.